UIKitDZ/UIKitDZ/View Controllers/BedViewController.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  FlatViewController.swift
+//  UIKitDZ
+//
+//  Created by Анастасия Козлова on 26.09.2022.
+//
+
+import UIKit
+/// BedViewController - контроллер выбор кроватей
+class BedViewController: UIViewController {
+    
+    private let chouseButton: UIButton = {
+        let button = UIButton(type: .custom)
+        button.setTitle("Выбрать", for: .normal)
+        button.setTitleColor(UIColor.black, for: .normal)
+        button.layer.cornerRadius = 25
+        button.contentMode = .center
+        button.titleLabel?.font = .systemFont(ofSize: 20)
+        button.backgroundColor = .tertiaryLabel
+        return button
+    }()
+    
+    private let imageViewRoom: UIImageView = {
+        let image = UIImageView()
+        image.layer.cornerRadius = 10
+        image.clipsToBounds = true
+        image.contentMode = .scaleAspectFill
+        return image
+    }()
+    
+    private let imageArray = [UIImage(named: "кровать1"),
+                      UIImage(named: "кровать2"),
+                      UIImage(named: "кровать3")]
+    
+    private let textLabel: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 27)
+        label.textColor = .black
+        label.contentMode = .center
+        label.text = "Выберите объект"
+        
+        return label
+    }()
+    
+    private let nameLabel: UILabel = {
+        let label = UILabel()
+        label.text = "BED"
+        label.font = .systemFont(ofSize: 30)
+        label.textColor = .black
+        label.contentMode = .center
+        return label
+    }()
+    
+    private let descriptionTextView: UITextView = {
+        let text = UITextView()
+        text.backgroundColor = .groupTableViewBackground
+        text.textColor = .black
+        text.contentMode = .center
+        text.isEditable = false
+        text.isSelectable = false
+        text.font = .systemFont(ofSize: 20)
+        return text
+    }()
+    
+    private let nameHouseArray = ["Neo", "Amal Grey", "Casablanca"]
```

Comment:
```suggestion
    private let names = ["Neo", "Amal Grey", "Casablanca"]
```
забываем слово array